paper/paper.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "MNE-ICALabel: Automatically annotating ICA components with ICLabel in Python"
+tags:
+  - Python
+  - MNE
+  - MEG
+  - EEG
+  - iEEG
+  - ICA
+  - EEGLab
+authors:
+  - affiliation: 1
+    name: Adam Li
+    orcid: 0000-0001-8421-365X
+  - affiliation: 2
+    name: Jacob Feitelberg
+    orcid: tbd
+  - affiliation: 3
+    name: Anand Saini
+    orcid: tbd
+  - affiliation: 3
+    name: Mathieu Scheltienne
+    orcid: tbd
+affiliations:
+- index: 1
+  name: Department of Computer Science, Columbia University, New York, United States
+- index: 2
+  name: Department of Biomedical Engineering, Johns Hopkins University, Baltimore, United States
+- index: 3
+  name: tbd
```

Comment:
```suggestion
  name: Human Neuroscience Platform, Fondation Campus Biotech Geneva, Geneva, Switzerland
- index 4
  name: Center for Neuroprosthetics, Institute of Bioengineering, Ecole Polytechnique Fédérale de Lausanne (EPFL), Geneva, Switzerland
```

I'd like to add both (3) and (4) to my affiliations as I've spent time from both jobs on this project.
I have to double-check the EPFL (4) affiliation, I've got a doubt if I should point to the Center for Neuroprosthetics or CIBM.